stdlib/os/__init__.pyi

```diff
@@ -726,6 +756,7 @@ if sys.platform != "win32":
         def plock(op: int) -> None: ...  # ???op is int?
 
 class _wrap_close(_TextIOWrapper):
+    def __init__(self, stream: _TextIOWrapper, proc: Popen[str]) -> None: ...
```

Comment:
This doesn't seem right, I don't think `os` will call into the subprocess module

Reply:
https://github.com/python/cpython/blob/695d47b51e3e197de5448a1eb2f618bef6d59ac8/Lib/os.py#L983

Reply:
Well I guess I was wrong :)